.github/workflows/publish.yaml

```diff
@@ -0,0 +1,54 @@
+name : Publish Webhook Images
+
+on:
+  push:
+    tags:
+      - release/v*
```

Comment:
Looking at the tags in webhook, there are no `release/` prefix: https://github.com/rancher/webhook/tags.

Reply:
Fixed